### ISSUE TITLE
[Hacktoberfest] Convert to typescript

### ISSUE THREAD

**Issue by jsdario** [netbeast/colorsys#17]
This task should not be very difficult since it's a single file library. I think we could gain significant value from it, achieving a lot of soundness in a pure functional library, with further mantainability for years to come.

**Comment by barthap** [comment]
Maybe converting to TypeScript is not necessary, but adding type declarations would be enough.
I made a PR #27 with `colorsys.d.ts` - my fork works pretty well with my TypeScript projects.

**Comment by jsdario** [comment]
Thanks for that

**Comment by jnguyenmt** [comment]
Did the version with the typescript file get published? I'm currently not seeing it in `v1.0.22 `

**Comment by barthap** [comment]
Yeah, it is just on github master, but npm package doesn't seem to be updated.

Temporarily, you can install it directly from github: `yarn add https://github.com/netbeast/colorsys.git` or `npm i --save https://github.com/netbeast/colorsys.git`
